testing/extended/_ssh.py

```
@@ -0,0 +1,103 @@
+    raw_target = target or os.environ.get("SSH_TARGET", "")
+    if not raw_target:
+        raise RuntimeError("SSH_TARGET is empty — required for ssh_run().")
+
+    user, host = _parse_ssh_target(raw_target)
+    command = shlex.join(argv)
+
+    client = paramiko.SSHClient()
+    # The dev environment is operator-owned and lives on a private network;
+    # we trust the host on first connect and let known_hosts pin subsequent
+    # ones. AutoAddPolicy is the right choice for a personal dev runbook.
+    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Ruff (0.15.14)

[error] Paramiko call with policy set to automatically trust the unknown host key

(S507)